src/fu-engine.c

```diff
@@ -8821,6 +8821,9 @@ fu_engine_constructed(GObject *obj)
 #endif
 
 	fu_context_add_compile_version(self->ctx, "org.freedesktop.fwupd", VERSION);
+#ifdef SOURCE_VERSION
+	fu_context_add_compile_version(self->ctx, "org.freedesktop.fwupd.source", SOURCE_VERSION);
```

Comment:
i think add a `if (g_strcmp(SOURCE_VERSION, VERSION) != 0)` here to avoid setting org.freedesktop.fwupd.source unless it's different.